docs/pages/admin-guides/access-controls/access-graph/ssh-keys-scan.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Discover insecure Access Patterns with Teleport Policy
+description: Describes how to enable SSH Keys Scanning using Teleport Policy and Access Graph.
+---
+
+Organizations often struggle to maintain a secure environment due to the lack of visibility into access patterns.
+Long-lived SSH keys, for example, can pose a significant security risk as they can be forgotten or mismanaged while
+still providing non-auditable access to critical systems.
+
+With Teleport Policy's Access Graph, you can gain insights on how SSH keys are used within your environment. By scanning
+all SSH authorized keys present on your servers and SSH Private Keys present on company managed laptops, Teleport Policy can
+provide a visual representation of the access patterns and help you enhance the security of your environment.
+
+This functionality gives you insights into the following areas:
+
+- Which servers have SSH Authorized Keys that could be used to bypass Teleport?
+- Which users have SSH Private Keys that grant access to SSH servers?
```

Comment:
Interesting - how does this work? Does tsh do the scanning and report back?

Reply:
The Teleport SSH service monitors and tracks users along with their authorized key files. The `tsh keys scan` command initiates a scanning process on each user's laptop to detect potential private keys. It then reports the public key fingerprints to Teleport. 

Teleport's access graph analyzes all authorized keys and private keys, matching their fingerprints to establish direct access paths. This process ensures that each pk that grants access to a specific user in a server is defined and trackable.

In the future, we want to expand scanning to servers as well and import everything including server-server relationships. i.e. describe how insecure bastion hosts are